test/k6/src/tests/instances.js

```diff
@@ -276,15 +286,125 @@ function TC10_SoftDeleteInstance(data) {
+    "instanceOwnerIdentifier" : instanceOwnerIdentifier,
+  };
+
+  const res = instancesApi.getInstances(data.orgToken, filters, options);
+
+  const dataBody = JSON.parse(res.body);
+  const instanciationResultSuccess = check(res, {
+    "TC11_GetInstances_ByPersonNumber: Get instance for party. Status is 200": (
+      r
+    ) => r.status === 200,
+    "TC11_GetInstances_ByPersonNumber: Instances exists for party.": 
+    dataBody.count > 0,
+  });
+  addErrorCount(instanciationResultSuccess);
+
+  if (dataBody.count > 0) { // If instance(s) exists
+    const firstInstance = dataBody.instances[0];
+    const personNumber = firstInstance.instanceOwner.personNumber;
+
+    const success = check(res, {
+      "TC11_GetInstances_ByPersonNumber: Get instance for party. Person number matches instanceOwner.personNumber":
+      personNumber === data.personNumber,
+    });
+    addErrorCount(success);
+  }
+}
+
+//TC12 - Get all instances for party looked up with an organisation number
+function TC12_GetInstances_ByOrgNumber(data) {
+  // Creating the instances against the organisation number
+  const instanciationOptions = {
+    "orgNumber": data.orgNumber,
+    "token": data.orgToken,
+    "partyId": data.orgPartyId,
+    "org": data.org,
+    "app": data.app,
+    "serializedInstance": serializedInstance,
+  };
+
+  const instanciationResult = instancesApi.postInstance(instanciationOptions);
+
+  const instanciationResultSuccess = check(instanciationResult, {
+    "TC12_GetInstances_ByOrgNumber: Create new instance for organisation number. Status is 201": (r) =>
+      r.status === 201,
+    "TC12_GetInstances_ByOrgNumber: Create new instance for organisation number. Instance Id is not null": (r) =>
+      JSON.parse(r.body).id != null,
+  });
+  addErrorCount(instanciationResultSuccess);
+
+  if (instanciationResultSuccess) {
+    const instanceId = JSON.parse(instanciationResult.body)["id"];
+
+    const instanceOwnerIdentifier = `Organisation:${data.orgNumber}`;
+    const filters = {
+      "appId": `${data.org}/${data.app}`,
+    };
+    const options = {
+      "instanceOwnerIdentifier" : instanceOwnerIdentifier,
+    };
+  
+    const res = instancesApi.getInstances(data.orgToken, filters, options);
+  
+    const dataBody = JSON.parse(res.body);
+
+    const hasInstances = check(res, {
+      "TC12_GetInstances_ByOrgNumber: Get instance for party. Status is 200": (
+        r
+      ) => r.status === 200,
+      "TC12_GetInstances_ByOrgNumber: Instances exists for party.":
+      dataBody.count > 0,
+    });
+    addErrorCount(hasInstances);
+  
+    if (dataBody.count > 0) { // If instance(s) exists
+      const firstInstance = dataBody.instances[0];
+      const orgNumber = firstInstance.instanceOwner.organisationNumber;
+      const instanceIdSplit = firstInstance["id"].split("/");
+  
+      const successResponse = check(res, {
+        "TC12_GetInstances_ByOrgNumber: Get instance for party. InstanceId has expected format":
```

Comment:
This check has already been executed in a previous test. No need to include here as well